orchagent/p4orch/tests/fake_portorch.cpp

```diff
@@ -21,8 +26,12 @@ PortsOrch::PortsOrch(DBConnector *db, DBConnector *stateDb, vector<table_name_wi
                         PORT_STAT_FLEX_COUNTER_POLLING_INTERVAL_MS, true),
       port_buffer_drop_stat_manager(PORT_BUFFER_DROP_STAT_FLEX_COUNTER_GROUP, StatsMode::READ,
                                     PORT_BUFFER_DROP_STAT_POLLING_INTERVAL_MS, true),
-      queue_stat_manager(QUEUE_STAT_COUNTER_FLEX_COUNTER_GROUP, StatsMode::READ,
-                         QUEUE_STAT_FLEX_COUNTER_POLLING_INTERVAL_MS, true)
+      queue_stat_manager(QUEUE_STAT_COUNTER_FLEX_COUNTER_GROUP, StatsMode::READ, QUEUE_STAT_FLEX_COUNTER_POLLING_INTERVAL_MS, false),
+      queue_watermark_manager(QUEUE_WATERMARK_STAT_COUNTER_FLEX_COUNTER_GROUP, StatsMode::READ_AND_CLEAR, QUEUE_WATERMARK_STAT_FLEX_COUNTER_POLLING_INTERVAL_MS, false),
```

Comment:
For PG/Queue watermark, READ_AND_CLEAR mode is set. Is this for all watermark poll?

Reply:
yes. for all.
we only have per-object counter mode for buffer pool counters.
for all other counter groups, like watermark, queue, etc, all objects of a group share a unified mode.
btw this is test stub code but in the production code it's the same. :)

Reply:
Agree, it is is in p4orch test code. But we have similar mode in portsorch.cpp :)